Tip support (#1620)
software-mansion/starknet.py: docs/migration_guide.rst

3. :meth:`Client.wait_for_tx` will no longer raise ``TransactionRejectedError``, see the method docs for details.
4. :meth:`Client.get_messages_status`: changed ``transaction_hash`` type from ``str`` to ``Hash``.

Tip Support
-----------

> Ability to pass tip for the transaction has been added to following methods.
> These methods will use the default value o ``0`` for tip if not provided.

Reviewer: ```suggestion
Ability to pass tip for the transaction has been added to following methods.
If ``tip`` is not provided, a default value of ``0`` will be used
```
